test/integration/mon/inspec/mon_spec.rb

```diff
@@ -0,0 +1,59 @@
+describe file('/etc/nagios/mysql.cnf') do
+  its('content') { should match(/user = monitor/) }
+  its('content') { should match(/password = ToJzwUyqQmyV4GgMVpz0/) }
+  its('mode') { should cmp 0600 }
+  its('owner') { should eq 'nrpe' }
+  its('group') { should eq 'nrpe' }
+end
+
+%w(
+  innodb
+  pidfile
+  processlist
+  replication-delay
+).each do |p|
+  describe file("/etc/nagios/nrpe.d/pmp-check-mysql-#{p}.cfg") do
+    its('content') do
+      should match(%r{command\[pmp-check-mysql-#{p}\]=\/usr/lib64/nagios/plugins/pmp-check-mysql-#{p}})
+    end
+  end
+  describe command("/usr/lib64/nagios/plugins/pmp-check-mysql-#{p}") do
+    its('stdout') { should match(/^OK/) }
+    its('exit_status') { should eq 0 }
+  end
+end
+
+describe file('/etc/munin/plugin-conf.d/mysql') do
```

Comment:
Please change this to an ini resource.

Reply:
For the two resources above this comment, I was able to convert to `ini` resource, but this one gives me a hiccup.


The actual file in the `mon` instance
```
[centos@moncentos7-freefood-turquoise-9j3mrpc ~]$ sudo cat /etc/munin/plugin-conf.d/mysql
[mysql*]
env.mysqluser monitor
env.mysqlpassword ToJzwUyqQmyV4GgMVpz0
```

According to the ini resource docs on InSpec's website, I need an alternative syntax like this:
```
describe ini('/etc/munin/plugin-conf.d/mysql') do
  its(['mysql*', 'env.mysqluser']) { should eq 'monitor' }
  its(['mysql*', 'env.mysqlpassword']) { should eq 'ToJzwUyqQmyV4GgMVpz0' }
end
```

However, test failed:
```
  INI /etc/munin/plugin-conf.d/mysql
     ∅  ["mysql*", "env.mysqluser"] should eq "monitor"
     
     expected: "monitor"
          got: nil
     
     (compared using ==)

     ∅  ["mysql*", "env.mysqlpassword"] should eq "ToJzwUyqQmyV4GgMVpz0"
     
     expected: "ToJzwUyqQmyV4GgMVpz0"
          got: nil
     
     (compared using ==)

```
Do you know why?

Reply:
I assume that ``*`` is messing with the regex. I tried escaping it but it didn't make any changes. I guess for this one let's stick with the file resource.